fhir-client/src/main/java/com/ibm/fhir/client/impl/FHIRClientImpl.java

```diff
@@ -452,10 +452,11 @@ public FHIRResponse searchAll(FHIRParameters parameters, boolean isPost, FHIRReq
         } else {
             endpoint = endpoint.path("_search");
             builder = endpoint.request(getDefaultMimeType());
+            builder = addRequestHeaders(builder, headers);
```

Comment:
addRequestHeaders - where does this come from what does it do?

Reply:
this is used for adding the tenant and datastore headers.